core/src/main/java/org/apache/accumulo/core/fate/user/UserFateStore.java

```diff
@@ -270,9 +271,10 @@ protected Stream<FateIdStatus> getTransactions(EnumSet<TStatus> statuses) {
       Scanner scanner = context.createScanner(tableName, Authorizations.EMPTY);
       scanner.setRange(new Range());
       RowFateStatusFilter.configureScanner(scanner, statuses);
-      TxColumnFamily.STATUS_COLUMN.fetch(scanner);
-      TxColumnFamily.RESERVATION_COLUMN.fetch(scanner);
-      TxInfoColumnFamily.FATE_OP_COLUMN.fetch(scanner);
+      // columns fetched from TxAdminColumnFamily
```

Comment:
```suggestion
      // columns fetched here must be in/added to TxAdminColumnFamily for locality group benefits
```
My point with the comment was to ensure if we fetch more columns here in the future that we move them to TxAdmin for the locality group benefits.